### ISSUE TITLE
Validate user agent for Chrome only

### ISSUE THREAD

**Issue by davola** [raffjones/chrome-admin-menu-fix#1]
function load_scripts()
{
    $ua = $_SERVER['HTTP_USER_AGENT'];
    if (strpos($ua,'Chrome/') > -1)
    {
        // Chrome found!
        // Need jQuery
        wp_enqueue_script('jquery');

```
    // Path to our simple script
    $js_path = plugins_url( 'js/chrome-menu-admin-fix.js', __FILE__ );

    // Load the simple script to click the collapse button twice
    wp_enqueue_script( 'chrome-menu-admin-fix', $js_path, array('jquery'), '1.0.0', true );
}
```

}

**Comment by heyfletch** [comment]
Good idea. Why didn't I think to just switch to Firefox?!

Created a PR here https://github.com/raffjones/chrome-admin-menu-fix/pull/2

**Comment by codewithfeeling** [comment]
Your pull request was merged earlier - thanks for helping.
